dlp/snippets/inspect/inspect_test.go

```diff
@@ -36,6 +40,10 @@ const (
 
 	ssnFileName = "fake_ssn.txt"
 	bucketName  = "golang-samples-dlp-test2"
+
+	jobTriggerId          = "dlp-job-trigger-unit-test-case-12345678"
```

Comment:
We should append a random string. Otherwise, when tests run in parallel, it will cause problems.

Reply:
Append uuid, while creating the trigger.